modules/metastorage/src/test/java/org/apache/ignite/internal/metastorage/server/AbstractCompactionKeyValueStorageTest.java

```diff
@@ -79,6 +92,10 @@ void setUp() {
         assertEquals(List.of(4, 6), collectRevisions(SOME_KEY));
     }
 
+    abstract boolean isPersistent();
```

Comment:
Why do you need this method?

Reply:
We have tests that check something after the storage is restarted. The test implementation is volatile and cannot work with snapshots. I was thinking about how to make the bypass of tests that require working with disks more elegant. Any suggestions?

Reply:
I just didn't see any usages of it, am I missing something?

Reply:
See `org.apache.ignite.internal.metastorage.server.AbstractCompactionKeyValueStorageTest#testRevisionsAfterRestart`